### ISSUE TITLE
Exercise 2c

### ISSUE THREAD

**Issue by MrTomRod** [MrTomRod/PatternRecognitionDeepAsleep#5]
- Optimize learning rate (typically in the range [0.001, 0.1]).

The script can test different learning rates.

- Optimize number of training iterations. Plot a graph showing the accuracy on the training set and the validation set, respectively, with respect to the training epochs.

Accuracy plotting works.

Rest: to do.

- Perform the random initialization several times and choose the best network during vali-
dation.

To do.
